### PR TITLE
Check for xfsprogs on plugin startup.

### LINKED ISSUE #37: needs xfsprogs dependency
This is simple, but I am not sure how to fix this in a golang project.

In driver.go there is 
 cmd = exec.Command("mkfs.xfs", device)
Although it sends information of failure to the log
l.logger.Err(fmt.Sprintf("Create: mkfs.xfs error: %s output %s", err, string(out)))
It only puts this on the console
 return resp(fmt.Errorf("Error partitioning volume"))

My system is debian jessie, so might be worth checking to see if xfsprogs are installed (or just check the executable).

It may save a number of distracting bug complaints...

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Fixes #37 

Signed-off-by: Shishir Mahajan <shishirm@nvidia.com>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -39,6 +39,10 @@ func main() {
 		logrus.SetLevel(logrus.DebugLevel)
 	}
 
+	if _, err := os.Stat("/sbin/mkfs.xfs"); os.IsNotExist(err) {
```
Comment (rhatdan) [inline main.go]: You should search for the path rather then hard code it.

exec.Lookpath("mkfs.xfs")

Comment (shishir-a412ed) [reply main.go]: Fixed.

**Comment by shishir-a412ed** [comment]
@rhatdan PTAL.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**